src/pages/profile/profile.jsx

```diff
@@ -5,6 +5,27 @@ import Identicon from '@atoms/identicons'
 import styles from '@style'
 import { useDisplayStore } from '.'
 import ParticipantList from '@components/collab/manage/ParticipantList'
+import { useCallback, useEffect, useState } from 'react'
+
+async function reverseRecord(address) {
+  const response = await fetch('https://api.tezos.domains/graphql', {
```

Comment:
I don't think this is standardized across the code base but we should use axios since I'm not sure we polyfill fetch yet (for older browsers)

Reply:
switched to axios with https://github.com/teia-community/teia-ui/pull/356/commits/d04ad54692850530506fcb1f3fd31d72f33a9fac